python/tests/test_cdh_utils.py

```diff
@@ -196,11 +190,12 @@ def test_cdh_sample_autodiscovered_locally():
 
 
 def test_file_not_found_in_good_dir():
-    assert cdh_utils.readDSExport(path="data", filename="models") == None
+    with pytest.raises(ValueError):
```

Comment:
I explicitly didn't want this to raise an error, by the way. The desired behaviour was for this _not_ to bug, but instead to return None. That way you don't have to catch that in the ADMDatamart class - and it's pretty obvious that if readDSExport didn't return anything, it didn't find the file